app/views/events/_table_badges.html.erb

```diff
@@ -1,30 +1,253 @@
+    <button type="button" class="btn btn-round-xs btn-hardmode btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_hard">Hard
+      Mode
+    </button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_hard" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Hard Mode Exists</h4>
+          </div>
+          <div class="modal-body">
+            <p>This scenario may played in a special "Hard Mode" where the risk is increased, but for no extra reward.
+              The only extra reward is being able to say you did it in Hard Mode.</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
 <% end %>
 <% if table.scenario.retired? %>
-    <span class="badge badge-retired">Retired</span>
+    <button type="button" class="btn btn-round-xs btn-retired btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_retired">Retired</button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_retired" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Retired Scenario</h4>
+          </div>
+          <div class="modal-body">
+            <p>This scenario has been retired from organized play. It normally is not available to be played for credit,
+              but will be for this event, unless marked as for "No Society Credit".</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
 <% end %>
 <% if table.core? %>
-    <span class="badge badge-core">Core</span>
+    <button type="button" class="btn btn-round-xs btn-core btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_core">Core</button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_core" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Core Campaign</h4>
+          </div>
+          <div class="modal-body">
+            <p>This table is being offered as a Core Campaign game. You must have a Core Campaign character in order to
+              play, unless you would like play a Core pregen.</p>
+          </div>
```

Comment:
Hey, we could add links to the guide and such where the core campaign or addition evergreen rules exist.

Reword: "This is a <link>Core Campaign game</link>. You must play with a Core Campaign character, or use a Core pregren.